src/App.jsx

```diff
@@ -14,57 +15,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [activeGood, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      {activeGood === '' ? (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          {activeGood} is selected
+          <button
+            data-cy="ClearButton"
+            type="button"
+            onClick={() => setGood('')}
```

Comment:
Create a helper function for reset logic
```suggestion
            onClick={reset}
```